data/root/etc/inst_setup

```diff
@@ -197,7 +190,7 @@ if grep -qi "^VNC:.*1" /etc/install.inf ; then
   if test "$ec" = "0" ; then
     (
     sleep 3
-    /usr/bin/slptool register "service:YaST.installation.suse:vnc://${hostname}:5901"
+    /usr/bin/slptool register "service:YaST.installation.suse:vnc://${host_name}:5901"
```

Comment:
ehm, ok. So why do you explicitly append it? I'm just curios, not blocker for me ;-)